techniques/general/G215.html

```diff
@@ -126,18 +127,18 @@ <h2>Example</h2>
 
 
 <section id="tests">
-	<h2>Tests</h2>
+	<h2>検証</h2>
 		<section class="test-procedure" id="procedure">
-		<h3>Procedure</h3>
-		<p>For content sliders that respond to path-based gestures:</p>
+		<h3>手順</h3>
+		<p>軌跡ベースのジェスチャに対応しているコンテンツスライダーにおいて:</p>
 		<ol>
-    		<li>Check that single pointer operable controls exist over or adjacent to the content slider that advance to adjacent chunks of content.</li>
+    		<li>シングルポインタによる操作が可能なコントロールが、当該コンテンツスライダーの上又はすぐ近くに存在しており、隣接するコンテンツのチャンクに進むことができることを確認する。</li>
```

Comment:
Check that ... の訳を、他のテクニックと合わせました。
```suggestion
    		<li>シングルポインタによる操作が可能なコントロールが、当該コンテンツスライダーの上又はすぐ近くに存在しており、隣接するコンテンツのチャンクに進むことができることをチェックする。</li>
```